test_views.py

```diff
@@ -18,36 +18,6 @@
 
 
 class AuthorizedViewsTests(AuthorizedTest):
-    def test_delete_bet(self):
-        bet = BetFactory.create()
-        share = bet.share
-
-        self.assertEqual(db.session.query(Bet).count(), 1)
-
-        response = self.app.delete(f"/api/v1/shares/{share.id}/bets/{bet.id}")
-
-        self.assertEqual(response.status_code, 204)
-        self.assertEqual(db.session.query(Bet).count(), 0)
-
-    def test_delete_bet_unkown_bet(self):
-        bet = BetFactory.create()
-        share = bet.share
-
-        self.assertEqual(db.session.query(Bet).count(), 1)
-
-        response = self.app.delete(f"/api/v1/shares/{share.id}/bets/{bet.id + 1}")
-
-        self.assertEqual(response.status_code, 404)
-
-    def test_add_share(self):
```

Comment:
We no longer have an endpoint to add shares. Instead, users create `members` which get associated with a (new) share automatically.